### PR TITLE
[#156224442] Add email address for security users

### PR DESCRIPTION
**Pull request by HawiCaesar**
### What does this PR do ?
- Allow the admin to add an email address for security users

### Description of work done
- change admin dashboard view to include email address
- remove code in `def save()`  that adds custom email address in security model
- refactor tests to include email address

### Screenshots if appropriate
screen shot 2018-03-28 at 11 54 11

### Relevant PT Story
[#156224442](https://www.pivotaltracker.com/story/show/156224442)

### CODE REVIEW

**Comment by alexkiura** [comment]
@HawiCaesar that's not necessary. We can add that file to .coveragerc to prevent it from being analysed.

File: api/models.py [hunk 1]
```diff
@@ -187,8 +187,4 @@ class Meta:
         verbose_name = "Security User"
 
     def save(self, *args, **kwargs):
```
Comment (egichuri) [inline api/models.py]: do we still need this?

Comment (HawiCaesar) [reply api/models.py]: I left it cause I guess you mentioned this `After discussions with the android team it seems like we'll not be implementing use of SMSs for now`

Comment (egichuri) [reply api/models.py]: yeah.. so i think we can get rid of the `save` methods since we don't have any more customizations.

Comment (HawiCaesar) [reply api/models.py]: Alright.

Comment (HawiCaesar) [reply api/models.py]: Done.

**Pull request merged**